quantities/astronomy.hpp

```diff
@@ -11,19 +11,21 @@ namespace principia {
 namespace quantities {
 
 // This namespace contains units commonly used in astronomy that are not
-// accepted for use with the SI.  See Résolution B3 "Sur les valeurs
-// recommandées de constantes de conversion pour une sélection de propriétés
-// solaires et planétaires" adopted at the XXIX General Assembly of the IAU in
-// 2015.
+// accepted for use with the SI.  See Résolution B2 "Re-définition de l’unité
```

Comment:
This comment is now inaccurate, since the au is accepted for use with the SI.

Change to 
```c++
// This namespace contains units commonly used in astronomy.
```

Group the definitions by resolution and cite the appropriate resolution there:
```C++
// Résolution B2 "Re-définition de l’unité astronomique de longueur" adopted
// at the XXVIIIth General Assembly of the IAU in 2012.
constexpr Length AstronomicalUnit      = 149597870700 * si::Metre;

// See note 4 of Résolution B2 "Sur la recommandation du "point zéro" des échelles
// de magnitude bolométrique absolue et apparente" adopted
// at the XXIXth General Assembly of the IAU in 2015.
constexpr Length Parsec                = 648000 / π * AstronomicalUnit;

// System of nominal solar and planetary conversion constants,
// Résolution B3 "Sur les valeurs  recommandées de constantes de
// conversion pour une sélection de propriétés solaires et planétaires" adopted
// at the XXIXth General Assembly of the IAU in 2015.
constexpr Mass   SolarMass             = 1.98855e30 * si::Kilogram;
// [...]
constexpr Length EarthEquatorialRadius = 6.3781e6 * si::Metre;
constexpr Length SolarRadius           = 6.957e8 * si::Metre;

constexpr Time   JulianYear            = 365.25 * si::Day;
constexpr Length LightYear             = constants::SpeedOfLight * JulianYear;
```

Note: this is `SolarRadius`, *not* `SolarEquatorialRadius`; the system of nominal solar and planetary conversion constants only distinguishes equatorial and polar radii for Earth and Jupiter, hence the notation ℛ<sup>N</sup><sub>eE</sub> and ℛ<sup>N</sup><sub>pE</sub> but ℛ<sup>N</sup><sub>☉</sub>; there seems to have been some confusion on [the relevant comment on #1846](https://github.com/mockingbirdnest/Principia/pull/1846/files/962472fb8bacc7c949859b8e7e99a54f123b83b5#r194225011).

The Julian year and corresponding light-year are not defined in any resolution that I know of, but there is no ambiguity as to their definition.

I have no idea where our `LunarDistance` comes from, and I would be happy to get rid of it; it seems to be used in two tests, and be irrelevant to both.